src/Resources/contao/library/Contao/Controller.php

```diff
@@ -174,12 +174,13 @@ public static function getTemplateGroup($strPrefix)
 	/**
 	 * Generate a front end module and return it as string
 	 *
-	 * @param mixed  $intId     A module ID or a Model object
-	 * @param string $strColumn The name of the column
+	 * @param mixed  $intId        A module ID or a Model object
+	 * @param string $strColumn    The name of the column
+	 * @param bool   $blnIgnoreEsi Whether ESI should be ignored
 	 *
 	 * @return string The module HTML markup
 	 */
-	public static function getFrontendModule($intId, $strColumn='main')
+	public static function getFrontendModule($intId, $strColumn='main', $blnIgnoreEsi = false)
```

Comment:
I think this should be reversed. For BC reasons, ESI should be disabled by default, and our core calls should enabled it if appropriate.